lib/matplotlib/transforms.py

```diff
@@ -682,6 +682,63 @@ def intersection(bbox1, bbox2):
 class Bbox(BboxBase):
     """
     A mutable bounding box.
+
+    Examples
+    --------
+    The default constructor takes the boundary "points" ``[[xmin, ymin],
+    [xmax, ymax]]``.
+
+        >>> example_box = Bbox([[1, 1], [3, 7]])
+        >>> example_box
```

Comment:
You can leave out assigning to a variable and save this line.

Reply:
I use it later?

Reply:
On one hand, I don't think these are actually executed as part of building the docs and I'm not sure that `example_box` is actually clearer that some stand-in name like `some_bbox` or by writing out `Bbox(...)` where it used farther down.

I think any of those three options are acceptable.

Reply:
I didn't see that it's used later. I'd prefer that you define the box in the example where you use it. That way each example stays self-contained and it's not even more code lines.